ui/components/app/wallet-overview/token-overview.test.js

```diff
@@ -54,6 +54,7 @@ describe('TokenOverview', () => {
     });
 
     it('should show ConvertTokenToNFT modal when token passed in props is an ERC721', () => {
+      process.env.COLLECTIBLES_V1 = true;
```

Comment:
Make sure to set this back to false after the test. Since this is a global, it may affect future tests (if you run the entire test suite).

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13352/commits/f4315ebf0f9226e465b9dcc123d36efbd1f682e1